### PR TITLE
User quota is not updated during authorization

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#434: User quota is not updated during authorization
2. ...

### Manual testing scenarios (*)
1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Click on "Sign in" link in top right corner and login with the Adobe Stock account with credits
5. Click on the logged-in user name (that appears instead of "Sign in" link) to show user profile popup

### Expected result (*)
Credits/Images are showing the default value "0", they are updated only after the page is refreshed

### CODE REVIEW

File: AdobeIms/Controller/Adminhtml/User/Profile.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 use Magento\Backend\App\Action;
 use Magento\Framework\Controller\ResultFactory;
 use Psr\Log\LoggerInterface;
+use Magento\AdobeStockClientApi\Api\ClientInterface;
```
Comment (sivaschenko) [inline AdobeIms/Controller/Adminhtml/User/Profile.php]: Unfortunately, we cannot use adobe stock modules from `AdobeIms` as it has to be a separate independent package.

I think the solution can be: 
 - moving User data generation out of Controller to a dedicated model and applying a plugin to this model from AdobeStockAsset module.
OR
 - updating the quota via additional asynchronous request after login (this is more preferable than plugin) - additional request should not affect user experience as it should be performed only once after login

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @Nazar65 thanks for the fix. Please take a look at my review comment!

File: AdobeIms/Controller/Adminhtml/User/Profile.php (hunk 1 above)
Comment (Nazar65) [reply AdobeIms/Controller/Adminhtml/User/Profile.php]: ok, i think second way is better. i will try to implement it.

**Comment by Nazar65** [comment]
hi @sivaschenko could you review new changes ?

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**